src/components/status-image/status-image.css

```
@@ -1,4 +1,21 @@
 .StatusImage {
+    
     border: 1px solid white;
-    background-color: var(--color-busy);
+    height: 423px;
+    width: 414px;
+}
+
+img {
+    position: absolute;
+    max-height: 423px;
+    max-width: 414px;
+    left: 0;
```

Position absolute is correct. DO NOT use absolute values here. The parent container already has those values. Use height and width 100% instead. Place the image at the bottom left corner (not only the left). The result will be the same however if you resize the image it will always be placed properly.
You could also try to add object fit cover for small screens.
```suggestion
    height: 100%;
    width: 100%;
    left: 0;
    bottom:0;
    object-fit: cover;
```